*** gss/src/main/java/org/globus/gsi/gssapi/GlobusGSSContextImpl.java	2018-07-31 17:42:58.507114834 +0000
--- gss/src/main/java/org/globus/gsi/gssapi/GlobusGSSContextImpl.java	2018-07-31 17:43:15.412046154 +0000
*************** done:      do {
*** 1306,1315 ****
              throw new GlobusGSSException(GSSException.FAILURE, e);
          }
  
- 	if (this.forceSSLv3AndConstrainCipherSuitesForGram.booleanValue())
-            this.sslEngine.setEnabledProtocols(GRAM_PROTOCOLS);
-         else
-            this.sslEngine.setEnabledProtocols(ENABLED_PROTOCOLS);
  	logger.debug("SUPPORTED PROTOCOLS: " +
                      Arrays.toString(this.sslEngine.getSupportedProtocols()) +
                      "; ENABLED PROTOCOLS: " +
--- 1306,1311 ----
